WFInfo/Main.cs

```diff
@@ -412,6 +432,13 @@ public static void FinishedLoading()
         }
         public static void UpdateMarketStatus(string msg)
         {
+            Debug.WriteLine($"New market status received: {msg}");
+            if (!UserAway)
```

Comment:
Status is only stored if the user is not away.  automated AFK status changes are irrelevant.